generate-stats

```diff
@@ -68,6 +68,9 @@ def main():
             links.append(urls[1][0])
         top10.append([row[0], row[1], links])
 
+    if not top10:
+        top10.append(["No failures in the last {0} days".format(opts.days), 0, []])
```

Comment:
This soo needs to add a 🍻 ! :grin:

Reply:
:smile: If we ever get to that state, lets do it! But now (and for some time) it will mean mostly (You picked too few days and we haven't ran any tests in that timeframe or your DB is too old).